terraform/sqs/alarms.tf

```diff
@@ -1,7 +1,3 @@
-resource "aws_sns_topic" "topic" {
-  name = "${aws_sqs_queue.dlq.name}_alarm"
-}
```

Comment:
Could we use the topic level sns module here instead? Would that get us what we need w/o forcing it out of the module?